mozilla/fx-private-relay
# MPP-3153 - add stripe.js

This PR fixes MPP-3153
 


# New feature description

To best leverage Stripe’s advanced fraud functionality, include this script on every page, not just the checkout page. This [allows Stripe to detect suspicious behavior](https://stripe.com/docs/disputes/prevention/advanced-fraud-detection) that may be indicative of fraud as customers browse your website.

# Screenshot (if applicable)

![image](https://github.com/mozilla/fx-private-relay/assets/3924990/629caa0d-261d-4ed4-89ed-5318d9a0e20b)

# How to test

Load the branch and inspect the page (any page). You should see the following: 

![image](https://github.com/mozilla/fx-private-relay/assets/3924990/771672ac-f2d1-47b4-a304-32b9774fbbed)


# Checklist (Definition of Done)
